zend_abstract_interface/properties/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+if(PHP_VERSION_DIRECTORY STREQUAL "php7" OR PHP_VERSION_DIRECTORY STREQUAL "php8")
+    set(PHP_PROPERTIES_VERSION_DIRECTORY "php7-8")
+else()
+    set(PHP_PROPERTIES_VERSION_DIRECTORY PHP_VERSION_DIRECTORY)
+endif()
+
+add_library(zai_properties "${PHP_PROPERTIES_VERSION_DIRECTORY}/properties.c")
+
+target_include_directories(zai_properties PUBLIC
+                                          $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
+                                          $<INSTALL_INTERFACE:include>)
+
+target_compile_features(zai_properties PUBLIC c_std_99)
```

Comment:
I was expecting this to be either `INTERFACE c_std_99 PRIVATE c_std_11` or `PUBLIC c_std_11` -- any specific reason for choosing C99?

Reply:
just copy paste from the other CMakeLists. Doesn't really matter as long as it compiles?